### PR TITLE
Remove unused imports

### PR DESCRIPTION
**Pull request by mauricioabreu**

### CODE REVIEW

**Comment by hangtwenty** [comment]
:+1:

File: tldextract/tldextract.py [hunk 1]
```diff
@@ -210,7 +208,7 @@ def __call__(self, url):
 
         if not tld and netloc and netloc[0].isdigit():
             try:
-                is_ip = socket.inet_aton(netloc)
```
Comment (john-kurkowski) [inline tldextract/tldextract.py]: Yay cleaning up unused variables! Boo this block getting more confusing what the side-effects are doing. I'll extract this into a function.

**Comment by john-kurkowski** [comment]
Thanks!

File: tldextract/tldextract.py (hunk 1 above)
Comment (john-kurkowski) [inline tldextract/tldextract.py]: Extracted in f1a83405f84cfb8c676f1485f1ecc6c64c07effa.

**Comment by mauricioabreu** [comment]
You are welcome! Since I started to use this package, I am probably going to refactor some parts of the codebase. :-)

**Pull request merged**